lib/AST/Type.cpp

```diff
@@ -3232,6 +3233,14 @@ Type ProtocolCompositionType::get(const ASTContext &C,
   return build(C, CanTypes, HasExplicitAnyObject);
 }
 
+StringRef
+AnyFunctionType::ExtInfo::Uncommon::printClangFunctionType(
+    ClangModuleLoader *cml, SmallVectorImpl<char> &buf) {
```

Comment:
This method should probably just print into a stream, and the place where you add quotes is the place that should print into a temporary string.